internal/controller/scheduler/scheduler.go

```diff
@@ -90,9 +96,12 @@ func (scheduler *Scheduler) Run() {
 
 func (scheduler *Scheduler) reportStats(workers []v1.Worker, vms []v1.VM) {
 	for _, worker := range workers {
+		workersLastSeen.With(map[string]string{"worker_name": worker.Name}).Set(float64(worker.LastSeen.Unix()))
```

Comment:
Do you want a duration between now and `LastSeen`?  Otherwise it seems to be ever growing metric.

Reply:
Interesting, I've not seen it done that way before. 
If you look at some other metrics, they will export the timestamp then you can subtract the current timestamp in your alert and set your own criteria for alerting on: 

i.e. 

```
(avg(nginx_ingress_controller_ssl_expire_time_seconds) by (host) - time()) < 604800
```

Reply:
That's an option. I just never used it like that. What do you think of still doing this as an "age" and call the metric `orchard_worker_last_seen`.

For example, we use GCP's Stackdrived and PromQL metrics are more expensive that just builtin ones. For similar setups having an age will simplify setting an alert.

Reply:
So instead of a timestamp, it's seconds since last seen?

Reply:
I think so, as per OpenTelemetry's [`Metrics Semantic Conventions`](https://opentelemetry.io/docs/specs/semconv/general/metrics/):

>When instruments are measuring durations, seconds (i.e. s) SHOULD be used.